plugins/context_datalayer_condition.inc

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * @file
+ * Context Datalayer condition plugin for Context API.
+ */
+
+/**
+ * Context Condition plugin to detect datalayer values.
+ */
+class context_datalayer_condition extends context_condition {
+  /**
+   * Implements options_form().
+   */
+  public function options_form($context) {
+    $form = parent::options_form($context);
+
+    $form['context_datalayer_condition']['data'] = array(
```

Comment:
This nonsense is because the form IDs are not unique between forms, resulting in the ID incrementing and _being lost_ when AJAX redraws the form.  This means the click event only happens once...